inc/composer/class-command.php

```diff
@@ -688,7 +688,7 @@ private function get_db_connection_data() {
 		$lines = exec( 'tput lines' );
 
 		$base_command = sprintf(
-			"$command_prefix %s exec -e COLUMNS=%d -e LINES=%d db",
+			"$command_prefix %s exec -e COLUMNS=%d -e LINES=%d -T db",
```

Comment:
Does this mess up the formatting of the output? I've found that disabling the pseudo-TTY allocation causes line breaks to collapse. Ref: https://github.com/docker/compose/issues/8833

Reply:
Originally when I created this I was on an old version of docker, however after updating the output from exec was no longer getting returned and only output using exec. Others at Nexstar also complained of the same issue. Disabling the pseudo-TTY is a workaround that was reported for local server to fix the issue. Since we are using a regex expression to pull out our values the formatting of the string should not matter.